pkg/apis/config/zz_generated.deepcopy.go

```diff
@@ -22,9 +22,9 @@ limitations under the License.
 package config
 
 import (
-	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	runtime "k8s.io/apimachinery/pkg/runtime"
-	v1 "k8s.io/kube-scheduler/config/v1"
+	apisconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"
```

Comment:
`apisconfig` is the generated abbreviate. Maybe we can use it to replace the `unversioned` in other files to keep consistent.

Reply:
And the defaultconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"